### ISSUE TITLE
Deprecated function Biostrings::fasta.info() - use fasta.seqlengths() instead

### ISSUE THREAD

**Issue by HenrikBengtsson** [HenrikBengtsson/aroma.seq#12]
When `print()`:ing a `FastaReferenceFile` we now (R 3.2.0) get:

``` r
Warning message:
'Biostrings::fasta.info' is deprecated.
Use 'fasta.seqlengths' instead.
See help("Deprecated")
```

**Comment by HenrikBengtsson** [comment]
Seems as if that be replaced straight off;

``` r
> fasta.info(getPathname(fa))
gi|9626243|ref|NC_001416.1| Enterobacteria phage lambda, complete genome
                                                                   48502
Warning message:
'fasta.info' is deprecated.
Use 'fasta.seqlengths' instead.
See help("Deprecated")
> fasta.seqlengths(getPathname(fa))
gi|9626243|ref|NC_001416.1| Enterobacteria phage lambda, complete genome
                                                                   48502
```

**Comment by HenrikBengtsson** [comment]
Fixed in commit f735c1dd.
